articles/macos-mdm-setup.md

```diff
@@ -53,7 +52,52 @@ If no default team is set for a host platform (macOS, iOS, or iPadOS), then newl
 
 > A host can be transferred to a new (not default) team before it enrolls. In the Fleet UI, you can do this under **Settings** > **Teams**.
 
-### Simple Certificate Enrollment Protocol (SCEP)
+## Volume Purchasing Program (VPP)
+
+> Available in Fleet Premium
+
+To connect Fleet to Apple's VPP, head to the guide [here](https://fleetdm.com/guides/install-vpp-apps-on-macos-using-fleet).
+
+### Best practice
+
+Most organizations need only one ABM MDM server and one VPP token to manage their devcices and content.
+
+Some organizations, like:
+
+- Managed Service Providers (MSP)
+- enterprises that acquire new business and as a result inherit new devices
+- unbrella organizations that preside over entities with separated purchasing authority (like a hospital or university) 
+
+may have have a need for multupe ABM and VPP connections.
+
+For an MSP, the best practice is to have one ABM and VPP connection per customer. 
+
+The default teams in Fleet for each client's ABM token in Fleet will look like this:
+- macOS: 💻 Client A - Workstations
+- iOS: 📱🏢 Client A - Company-owned iPhones
+- iPadOS:🔳🏢 Client A - Company-owned iPads
+
+Client A's VPP token will be assigned to the above teams.
+
+For enterprises that acquire, the best practice is to add a new ABM and VPP connection for each acquisistion.
```

Comment:
For enterprises that acquire, the best practice is to add a new ABM and VPP token for each acquisition.